docs/content/guides/6.multistore/3.patterns/2.design/4.different-code-per-store.md

```diff
@@ -5,4 +5,254 @@ navigation:
+- To adjust styling for specific stores or brands without altering the functionality.
+
+### The Solution: Component Override
+
+In cases where a specific store requires customization, you can override the component by duplicating and modifying it within that store's directory.
+
+#### Example Workflow
+
+1. **Duplicate** the component inside the store-specific directory.
+2. **Modify** the duplicated component as per the store's requirements (e.g., layout, structure, or styles).
+
+#### In Next.js
+
+Here’s how to override `ExampleComponent` for a specific store (`store-a`) in a Next.js project:
+
+```tsx
+// /apps/stores/store-a/storefront-unified-nextjs/components/example-component.tsx
+
+// Props remain unchanged
+interface ExampleComponentProps {
+  title: string;
+  imageSrc: string;
+}
+
+export default function ExampleComponent({ title, imageSrc }: ExampleComponentProps) {
+  return (
+    <div className="flex items-center w-md"> {/* Customized className for store-a */}
+      <img src={imageSrc} alt={title} />
+      <h2>{title}</h2>
+      {/* Customized content for store-a */}
+    </div>
+  );
+}
+```
+
+#### In Nuxt
+
+The process is similar in Nuxt. Here’s an example:
+
+```vue
+// /apps/stores/store-a/storefront-unified-nuxt/components/ExampleComponent/ExampleComponent.vue
+
+<template>
+  <!-- Customized className for store-a -->
+  <div class="flex items-center w-md">
+    <img :src="imageSrc" :alt="title" />
+    <h2>{{ title }}</h2>
+    <!-- Customized content for store-a -->
+  </div>
+</template>
+
+<script setup lang="ts">
+defineProps<{
+  title: string;
+  imageSrc: string;
+}>();
+</script>
+```
+
+### Outcome
+
+In both cases (Next.js and Nuxt), props remain untouched, ensuring compatibility with the rest of the application. Adjustments remain restricted to layout or presentation, avoiding unnecessary duplication or breaking changes to functionality.
+
+## Breaking Changes in Component Markup
+
+Some changes may require adjustments to the component’s **props interface**, meaning that the component is no longer compatible with its previous usage. These are referred to as **breaking changes** because they can potentially impact other parts of the application if not handled correctly.
+
+### When to Use
+
+- When introducing additional props for brand-specific requirements.
+- When removing unused or deprecated props for cleaner, leaner implementations.
+- When creating a fundamentally different component design unique to specific brands or stores.
+
+### Example Workflow for Breaking Changes
+
+#### Scenario 1: Adding Props
+
+The `ExampleComponent` requires a new prop `customSubtitle`, exclusively for `store-b`.
+
+1. **Duplicate** the component for `store-b`.
+2. **Modify the props interface** to include the new or modified prop.
+3. **Implement fallback behaviors** to ensure existing components using the old interface remain functional, or document the change clearly to propagate updates across the codebase.
```

Comment:
This is too vague. Can't we state here that the new props should be optional?

Reply:
I've changed the description